packages/@n8n/config/src/index.ts

```diff
@@ -97,4 +99,7 @@ export class GlobalConfig {
 
 	@Nested
 	multiMainSetup: MultiMainSetupConfig;
+
+	@Nested
+	security: SecurityConfig;
```

Comment:
we don't need to add everything in the global config. we can move the config class to core as well.

Reply:
Moved it to core. But wouldn't it make sense to have all the app configuration in a single location instead of being spread around the codebase?